app/src/main/java/org/oppia/app/player/exploration/ExplorationManagerFragmentPresenter.kt

```diff
@@ -0,0 +1,49 @@
+package org.oppia.app.player.exploration
+
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.Observer
+import androidx.lifecycle.Transformations
+import org.oppia.app.fragment.FragmentScope
+import org.oppia.app.model.Profile
+import org.oppia.app.model.ProfileId
+import org.oppia.app.model.StoryTextSize
+import org.oppia.domain.profile.ProfileManagementController
+import org.oppia.util.data.AsyncResult
+import org.oppia.util.logging.Logger
+import javax.inject.Inject
+
+/** The presenter for [ExplorationManagerFragment]. */
+@FragmentScope
+class ExplorationManagerFragmentPresenter @Inject constructor(
+  private val profileManagementController: ProfileManagementController,
+  private val logger: Logger,
+  private val activity: AppCompatActivity
+) {
+  private lateinit var profileId: ProfileId
+
+  fun handleCreate(internalProfileId: Int) {
+    this.profileId = ProfileId.newBuilder().setInternalId(internalProfileId).build()
+    getProfileData().observe(activity, Observer<StoryTextSize> { result ->
+      (activity as DefaultFontSizeStateListener).onDefaultFontSizeLoaded(result)
+    })
+  }
+
+  private fun getProfileData(): LiveData<StoryTextSize> {
```

Comment:
Is this method name correct? Perhaps instead: ``retrieveStoryTextSize`` would be clearer.

Reply:
renamed to  retrieveStoryTextSize